Userland/Libraries/LibVT/Terminal.h

```diff
@@ -443,6 +444,10 @@ class Terminal : public EscapeSequenceExecutor {
     Optional<u16> m_column_before_carriage_return;
     bool m_controls_are_logically_generated { false };
     CursorKeysMode m_cursor_keys_mode { Cursor };
+
+    CharacterSetTranslator m_character_set_translator {};
+    size_t m_active_working_set_index { 0 };
```

Comment:
Does anything ever modify this?

Reply:
Not yet - there are some as of yet unimplemented sequences like Ctrl+O or Ctrl+N that should write to this variable.